forms-flow-web/src/components/Form/EditForm/FormEdit.js

```diff
@@ -980,7 +980,9 @@ const EditComponent = () => {
                 <div className="d-flex align-items-center justify-content-between">
                   <BackToPrevIcon onClick={backToForm} />
                   <div className="mx-4 editor-header-text">
-                    {formData.title}
+                    {formData.title.length > 90
+                    ? `${formData.title.slice(0, 90)}...` 
```

Comment:
already one function is there, textTruncate in helper, please use it

Reply:
added